app/presenters/account_reset/pending_presenter.rb

```diff
@@ -0,0 +1,21 @@
+module AccountReset
+  class PendingPresenter
+    include ActionView::Helpers::DateHelper
+
+    attr_reader :account_reset_request
+
+    def initialize(account_reset_request)
+      @account_reset_request = account_reset_request
+    end
+
+    def time_remaining_until_granted
+      distance_of_time_in_words(
+        Time.zone.now,
```

Comment:
ditto, if we make `now` a default parameter we can remove the `Timecop.freeze` in specs:

```ruby
def time_remaining_until_granted(now: Time.zone.now)
```